homeworks/vladislav.buzunar_buzyaka/4-objects/objects.js

```diff
@@ -0,0 +1,44 @@
+const tamagochi = {
+    name: 'Tamagochi',
+    age: 23,
+    healthLevels: 5,
+    happyLevels: 3,
+    foodLevels: 4,
+    noEating() {
+        tamagochi.healthLevels -= 1;
+        tamagochi.happyLevels -= 1;
+        tamagochi.foodLevels -= 1;
+
+        const tamagochiStates = `Здоров'я  в ${tamagochi.name} - ${tamagochi.healthLevels} Щастя - ${tamagochi.happyLevels} Ситість - ${tamagochi.foodLevels}`;
```

Comment:
This code is duplicated. You can try to create a separate method with it or try to create a getter.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get